core/templates/dev/head/pages/about-page/about-page.module.ts

```diff
@@ -32,11 +32,11 @@ import { HttpClientModule } from '@angular/common/http';
 })
 export class ServiceBootstrapComponent {}
 
-import { AppConstants } from 'app.constants';
+import { AppConstants } from '../../app.constants'
 import { InteractionsExtensionsConstants } from
-  'interactions/interactions-extension.constants';
+  '../../../../../../extensions/interactions/interactions-extension.constants';
```

Comment:
@NoumanMasood, Can you please explain the reason behind these changes?

Also, we don't expect a PR to be made from the develop branch, would you mind closing this PR and creatinga new one following the guideline provided in [this wiki page](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)?

Reply:
ok I will make new PR